modules/run-vault/run-vault

```diff
@@ -144,7 +166,24 @@ EOF
     consul_storage_type="ha_storage"
   fi
 
-  local readonly consul_storage=$(cat <<EOF
+
+  if [[ "$enable_dynamo" == "true" ]]; then
+    consul_storage=""
```

Comment:
Please declare `local consul_storage` and `local vault_storage` up above this `if`/`else` block. That way, both default to empty strings, and only one gets set to a proper value.

Reply:
Good call.  Adjusting.